app/assets/javascripts/components/NewsletterSticky.js

```diff
@@ -66,10 +74,11 @@ define([
 
     if (isActive) {
       this.closeButton.on('click', $.proxy(this._handleCloseClick, this));
+      this._firstFocusElement.on('keydown', $.proxy(this._handleFirstElementTab, this));
+      this._lastFocusElement.on('keydown', $.proxy(this._handleLastElementTab, this));
       this.$el.on('click', this.config.buttonDoneClass, $.proxy(this._handleCloseClick, this));
       $newsletterStickForm.on('ajax:error', $.proxy(this._formAjaxErrorHandler, this));
-    }
-    else {
+    } else {
       this.closeButton.off('click', $.proxy(this._handleCloseClick, this));
```

Comment:
I missed this first time round... We need to turn the handlers off to free up memory...

```this._firstFocusElement.off('keydown', $.proxy(this._handleFirstElementTab, this));
this._lastFocusElement.off('keydown', $.proxy(this._handleLastElementTab, this));```